### DISCUSSION TITLE
Need Help Resizing LXC Container for Pi-Hole

### DISCUSSION THREAD

**Discussion by Photo64** [tteck/Proxmox#563] [Q&A] [answered]
Was hoping someone might be able to guide me resizing an LXC container in Proxmox. I used tteck's script for creating an LXC container for Pi-Hole. I used the default setup of the 2gb size however after running it I keep getting a message that I am running out of disk space. Every day now I go in and delete the pi-hole.FTL.db to gain more space which seem to work at least for a day but now even that doesn't seem to be eliminating enough space. I also changed the MAXDBDAYS amount but again it doesn't seem to be providing enough space. Unfortunately I haven't found out any other way to help reduce space in Pi-hole so my current thought is to increase the space size of the disk. 

I see an option in Proxmox for the container under Resources->Root DIsk-> Volume Action to resize the disk. I have not found any information through searching if this will work. Would anyone know if I can change the disk size using this option or does it have to be done a different way or should I create a new container with an increased disk size?

**Comment by tteck** [comment, marked as answer]
That's the way to increase the size. Enter the amount (GB) to ADD to the disk.

**Reply by Photo64** [reply]
Thanks, that did it. ()
  Size of logical volume pve/vm-103-disk-0 changed from 2.00 GiB (512 extents) to 8.00 GiB (2048 extents).
  Logical volume pve/vm-103-disk-0 successfully resized.
resize2fs 1.46.5 (30-Dec-2021)
Filesystem at /dev/pve/vm-103-disk-0 is mounted on /tmp; on-line resizing required
old_desc_blocks = 1, new_desc_blocks = 1
The filesystem on /dev/pve/vm-103-disk-0 is now 2097152 (4k) blocks long.
